lte/cloud/go/services/nprobe/encoding/record_test.go

```diff
@@ -29,47 +30,41 @@ type timeTest struct {
 
 var (
 	generalizedTimeTestData = []timeTest{
-		{"2021-02-18T05:13:26.019519+00:00", true, "010000000ed7bfee760129d6180000"},
+		{"2021-02-18T05:13:26.019519+00:00", true, "32303231303231383035313332362e303139"},
 	}
 
 	// This is a minimal bearer activation record that should be parsed correctly.
 	encodedRecord = []byte{
```

Comment:
Here and elsewhere: for big values like this, instead of storing this value inline, move it to a a base64-encoded [golden file](https://softwareengineering.stackexchange.com/questions/358786/what-is-golden-files)

Reply:
I moved both tests to golden files